Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Build Stage
+FROM golang:1.21 AS build-stage
```

Comment:
Should this be `golang:1.21-alpine`?

Reply:
[9e1dfd0](https://github.com/mrjoelkamp/tuf-pk-ledger/pull/5/commits/9e1dfd0ffdf1d46f76efb1a243b9d94470f361a7)